llm_on_ray/inference/api_server_openai.py

```diff
@@ -71,13 +71,21 @@ def router_application(deployments, model_list, max_ongoing_requests):
     return RouterDeployment.bind(merged_client)
 
 
-def openai_serve_run(deployments, model_list, host, route_prefix, port, max_ongoing_requests):
+def openai_serve_run(deployments, model_list, host, route_prefix, port, max_ongoing_requests, new_deployment):
     router_app = router_application(deployments, model_list, max_ongoing_requests)
+    
+    if new_deployment:
+        for model_id, infer_conf in model_list.items():
+            print("deploy model: ", model_id)
+            route_prefix = infer_conf.route_prefix
+            deployment_name = infer_conf.name
```

Comment:
Since multiple models can be deployed at a time under http://localhost:8000/{route_prefix}/v1, we cannot iterate over model_list to get route_prefix  and deployment_name.

Reply:
Will revert this change, instead user can specify "openai_route_prefix" if they want to deploy to another prefix